vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -47,9 +47,9 @@ def __init__(self, list_queues, remove_queue):
 
     def updater_loop(self):
         while self.run_daemon_loop:
-            r_ports = self.pq.get_remove_tasks()
-            if r_ports:
-                remover(self.pq, r_ports)
+            dvs, r_ports = self.pq.get_remove_tasks()
+            if dvs and r_ports > 0:
+                remover(dvs, r_ports)
```

Comment:
I didn't find changes for remover function. Don't forget update remover() function =))))